### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by Gala-G**

### CODE REVIEW

File: maker-core/cvc.md [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
+Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Aligned Delegates to direct the evolution of the Maker Protocol.
 
-{% hint style="warning" %}
-This documentation describes functionality and processes that MakerDAO is in the process of implementing. Be aware that parts may be inaccurate or out of date.
-{% endhint %}
+{% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}
 
 ## Overview
+Each AVC produces Aligned Governance Strategy and Aligned Scope Artifact documents to update the [Scope Artifacts](list-of-scopes.md). This set of documents provides general guidelines for how [Aligned Delegates (ADs)](delegates.md) that follow their governance strategy should vote.
```
Comment (astronautthis) [inline maker-core/cvc.md:8]:
```suggestion
Each AVC produces an Aligned Governance Strategy document and Aligned Scope Artifact documents to update the Scope Artifacts of each [Scope]](list-of-scopes.md) once every quarter. The Aligned Governance Strategy document provides general guidelines for how Aligned Delegates (ADs) that follow their governance strategy should vote.
```

File: maker-core/cvc.md [hunk 2]
```diff
@@ -1,41 +1,42 @@
-# Constitutional Voter Committees (CVCs)
+# Aligned Voter Committees (AVC)
 
-Constitutional Voter Committees (CVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Constitutional Delegates to direct the evolution of the Maker Protocol.
+Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Aligned Delegates to direct the evolution of the Maker Protocol.
```
[the remark this answers is not in the document]
Comment (astronautthis) [reply maker-core/cvc.md:3]: Minor comment that the AVC only needs one MKR holder, so it isn't a group of MKR holders.

File: maker-core/list-of-scopes.md [hunk 3]
```diff
@@ -1,32 +1,14 @@
 # List of Scopes
 
-The Endgame Scopes are divided into three categories, Primary, Supporting, and Governance.
+{% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}
 
-{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+Here is a list of the five Scopes:
```
Comment (astronautthis) [inline maker-core/list-of-scopes.md:5]:
```suggestion
The Maker Protocol has five Scopes. They are:
```

File: maker-core/scopes-and-frameworks.md [hunk 4]
```diff
@@ -24,14 +19,13 @@ Any change made to a Scope Framework must aim to improve one or more of the foll
 - Simplicity of protocol
 
 ## Governance
-Each Scope has an Advisory Council. This Council consists of members who must fulfill certain requirements and have been approved by Maker Governance. The Council carries out advisory work related to improving the content of the Scope Framework.
-
-The exception to this is the Arbitration Scope, which does not have an Advisory Council.
+Each Scope has a Scope Advisory Council. This Council consists of members who must fulfill certain requirements and have been approved by Maker Governance. The Council carries out advisory work related to improving the content of the Scope Artifacts.
```
Comment (astronautthis) [inline maker-core/scopes-and-frameworks.md:22]: Can we add a link to AC and create an empty page for them?

File: maker-core/cvc.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Gala-G) [reply maker-core/cvc.md:3]: @astronautthis I believe all members must hold MKR. I asked Patrick, will let you know.

[the remark this answers is not in the document]
Comment (Gala-G) [reply maker-core/cvc.md:3]: What do you think about:

Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared strategic vision for the Maker protocol. They have two main tasks in governance:
1. To ensure that all [Aligned Delegates (ADs)](delegates.md) that are affiliated with them, execute the AVC's strategic vision when voting on governance issues.   
2. To propose modifications to the Scope Artifacts to strengthen (solidify? ossify?) them over time.

ty

[the remark this answers is not in the document]
Comment (blimpa) [reply maker-core/cvc.md:3]: Presumably non-Aligned VCs also share a strategic vision?

File: maker-core/cvc.md [hunk 5]
```diff
@@ -1,41 +1,42 @@
...
-{% endhint %}
+{% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}
 
 ## Overview
+Each AVC produces Aligned Governance Strategy and Aligned Scope Artifact documents to update the [Scope Artifacts](list-of-scopes.md). This set of documents provides general guidelines for how [Aligned Delegates (ADs)](delegates.md) that follow their governance strategy should vote.
 
-Each CVC produces Governance Strategy Position documents and Scope Framework Position Documents to update the [Scope Frameworks](list-of-scopes.md). This set of documents provides general guidelines for how [Constitutional Delegates (CDs)](delegates.md) that follow their governance strategy should vote.
+The documents produced by the AVCs abstract away the minutiae of day-to-day governance. Since Aligned Delegates execute AVC strategies, they allow MKR Holders to be reasonably informed and secure with regard to the general direction of the chosen delegate.
```
Comment (blimpa) [inline maker-core/cvc.md:10]: They = the documents, ADs, AVCs?

File: maker-core/cvc.md [hunk 6]
```diff
@@ -1,41 +1,42 @@
...
 
-Position Documents abstract away the minutiae of day-to-day governance. Since Constitutional Delegates execute CVC strategies, they allow MKR Holders to be reasonably informed and secure with regard to the general direction of the chosen delegate.
+According to their own strategy, AVCs fall somewhere in the spectrum that goes from **Hawk** to **Dove**:
+  - Hawkish AVCs focus more on short-term income and asset accumulation, tending to prefer higher spreads and lower budgets
+  - Dovish AVCs focus more on long-term growth and innovation, tending to prefer lower spreads and higher budgets
+  - Balanced AVCs try to find a middle ground between the two
+
+The Hawk, Dove, Balanced is a continuous spectrum, so many different hawkish or dovish AVCs can exist that are more or less Balanced.
```
Comment (blimpa) [inline maker-core/cvc.md:17]:
```suggestion
The Hawk/Balanced/Dove is a continuous spectrum, so many different hawkish or dovish AVCs can exist that are more or less Balanced.
```

File: maker-core/cvc.md [hunk 7]
```diff
@@ -1,41 +1,42 @@
...
-- At least one registered member must be a CC and a verified MKR holder
-- For each quarterly governance cycle, CVCs must:
-   1. Create a Governance Strategy Position and 12 Scope Framework Position documents
-   2. Hold one CVC Subcommittee meeting for each scope
+To become Active AVCs, AVCs are subject to additional eligibility requirements. For each quarterly governance cycle they must:
+   - Create one Aligned Governance Strategy document and five Aligned Scope Proposals
+   - Hold two AVC Subcommittee meetings for each scope
 
-The [Arbitration Scope](https://mips.makerdao.com/mips/details/MIP113) recognizes eligible CVCs. It can revoke this recognition anytime the CVC fails to meet all eligibility requirements.
+The [Governance Scope](https://mips.makerdao.com/mips/details/MIP113) recognizes Active AVCs. It can revoke this recognition anytime the AVC fails to meet all eligibility requirements.
```
Comment (blimpa) [inline maker-core/cvc.md:27]: (Not sure what "recognizes" means here)

File: maker-core/cvc.md [hunk 8]
```diff
@@ -1,41 +1,42 @@
...
 
 ## Member Participation Rewards
+Member Participation Rewards is a compensation scheme available to AVC members. It considers their participation in each scope.
```
Comment (blimpa) [inline maker-core/cvc.md:30]:
```suggestion
Member Participation Rewards is a compensation scheme available to AVC members. Rewards are distributed per scope.
```

File: maker-core/cvc.md [hunk 9]
```diff
@@ -1,41 +1,42 @@
...
+To be eligible for compensation, AVC members must:
+ - Attend the two AVC Subcommittee Meetings
```
Comment (blimpa) [inline maker-core/cvc.md:32]:
```suggestion
 - Attend the two quarterly AVC Subcommittee Meetings for a given scope
```

File: maker-core/list-of-scopes.md (hunk 3 above)
Comment (blimpa) [reply maker-core/list-of-scopes.md:5]: We're being inconsistent with the capitalization of "scopes".

"Scopes" seems better

File: maker-core/scopes-and-frameworks.md [hunk 10]
```diff
@@ -1,21 +1,16 @@
...
 
-## Modification of Scope Frameworks
-Scope Frameworks are modifiable within the constraints of Constitutional Boundaries that are immutably defined in the Maker Constitution. An ossification rule is also applied to elements of Scope Frameworks - broadly speaking, an element of a Scope Framework cannot undergo large changes if it has been unchanged for a relatively long period of time. There exist soft clauses that are not required to follow the ossification rule. 
+## Modification of Scope Artifact
+Scope Artifacts are modifiable within the constraints of Scope Boundaries that are immutably defined in the Maker Atlas. An ossification rule is also applied to elements of Scope Artifacts - broadly speaking, an element of a Scope Artifact cannot undergo large changes if it has been unchanged for a relatively long period of time. There exist soft clauses that are not required to follow the ossification rule.
```
Comment (blimpa) [inline maker-core/scopes-and-frameworks.md:11]: Not sure what is meant by "broadly" here

File: subdaos/overview.md [hunk 11]
```diff
@@ -1,11 +1,11 @@
 # SubDAOs
```
Comment (blimpa) [inline subdaos/overview.md:1]: What is this doing here lol

**Review by blimpa — CHANGES_REQUESTED** [review]

File: maker-core/cvc.md (hunk 7 above)
Comment (Gala-G) [reply maker-core/cvc.md:27]: https://mips.makerdao.com/mips/details/MIP101#2-5-3-aligned-voter-committee-recognition-process

**Comment by Gala-G** [comment]
Closing because needs fixes. Will do new PR after staging.  @blimpa edited documents are on mipeditors hackmd. Thanks

**Pull request closed without merging**